homeassistant/components/igloohome/__init__.py

```diff
@@ -0,0 +1,36 @@
+"""The igloohome integration."""
+
+from __future__ import annotations
+
+from igloohome_api import Api, Auth
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_CLIENT_ID, CONF_CLIENT_SECRET, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+PLATFORMS: list[Platform] = [Platform.SENSOR]
+
+type IgloohomeConfigEntry = ConfigEntry[Api]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: IgloohomeConfigEntry) -> bool:
+    """Set up igloohome from a config entry."""
+
+    session = async_get_clientsession(hass)
+    authentication = Auth(
+        session=session,
```

Comment:
```suggestion
    authentication = Auth(
        session=async_get_clientsession(hass),
```
no need for that var